web_programming/random_anime_character.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import requests
+from bs4 import BeautifulSoup
+from fake_useragent import UserAgent
+
+headers = {"UserAgent": UserAgent().random}
+URL = "https://www.mywaifulist.moe/random"
+
+
+def save_image(image_url: str, image_title: str) -> None:
+    """
+    Saves the image of anime character
+    """
+    image = requests.get(image_url, headers=headers)
+    with open(image_title, "wb") as file:
+        file.write(image.content)
+
+    return None
```

Comment:
```suggestion
```
This can safely be deleted.  We would only use `return None` on functions that could return other values.